axlearn_inference.py

```diff
@@ -267,7 +267,7 @@ def assert_top_p_allclose(x, y, p=0.99):
     print("max top p relative difference", np.max(rdiff))
     print("mean top p relative difference", np.mean(rdiff))
 
-    assert np.allclose(top_x, top_y, atol=1e-3, rtol=1e-2)
+    np.testing.assert_allclose(top_x, top_y, atol=1e-3, rtol=2e-2)
```

Comment:
we need to review the atol and rtol values and compare them against the smallest union of topP and topK on both x and y logits